### PR TITLE
Beginning work on Python section

### PR DESCRIPTION
**Pull request by Leslie-C**
Beginning "import" of python section and updating to Python 3.11. Will come back to add in type annotations.

### CODE REVIEW

File: part2-06-python-functions.Rmd [hunk 1]
```diff
@@ -4,7 +4,7 @@
 >
 >~Donald Knuth
 
-Functions (sometimes called “subroutines”) are arguably the most important concept in programming. We’ve already seen their use in many contexts, for example, when using functions like `len()` and `float()`. Here’s a bit of code that computes the GC content of a DNA sequence in a variable called `seq`:
+Functions (sometimes called "subroutines") are arguably the most important concept in programming. We’ve already seen their use in many contexts, for example, when using functions like `len()` and `float()`. Here’s a bit of code that computes the GC content of a DNA sequence in a variable called `seq`:
```
Comment (oneilsh) [inline part2-06-python-functions.Rmd:7]: We can maybe leave out the subroutines bit, that text was from back when I was trying to convert perl acolytes ;)

Comment (Leslie-C) [reply part2-06-python-functions.Rmd]: Ha! Sounds good, updated in 5762f2aad39534e02a45a7df96d15fa88c1d2dba

**Review by oneilsh — APPROVED** [review]

File: part1-02-logging-in.Rmd [hunk 2]
```diff
@@ -54,7 +54,7 @@ In any case, you will next be asked to enter your password. Note that *as you ty
             line-numbers 
             linkable-line-numbers">
 <code>
-oneils@atmosphere ~$
+[oneils@mbp ~]$
```
[the remark this answers is not in the document]
Comment (Leslie-C) [reply part1-02-logging-in.Rmd:57]: Happy to swap out as people prefer.

+1 on this being a good exercise to build GitHub skills - I don't see a "review". Am I missing it? Or is it just that there's now a Resolve conversation button?

[the remark this answers is not in the document]
Comment (oneilsh) [reply part1-02-logging-in.Rmd:57]: Yeah I think this thread is the review? Lol. Are you able to do the resolve conversation button?

**Pull request merged**